### ISSUE TITLE
Web accessibility - Image alternative text is not present

### ISSUE THREAD

**Issue by natnael2012** [learningequality/ka-lite#3298]
Branch - Develop

Expected Behavior - Each image must have an alt attribute. Without alternative text, the content of an image will not be available to screen reader users or when the image is unavailable. 
Standards and Guidelines
•	Section 508 (a)  http://webaim.org/standards/508/checklist#standarda
•	1.1.1 Non-text Content (Level A) http://webaim.org/standards/wcag/checklist#sc1.1.1

Current Behavior - Image alternative text is not present on the center and CC image found on footer section.

Snapshot of the display inspected by Wave checker
![alttext](https://cloud.githubusercontent.com/assets/8178368/6651121/2adc07ea-c9f2-11e4-92b0-80880a084664.png)

**Comment by radinamatic** [comment]
should be addressed with  #3292 PR

**Comment by MCGallaspy** [comment]
Landing page has changed, can someone revisit this issue and determine if something can/needs to be done?

**Comment by aronasorman** [comment]
Let's close this now, and reopen for accessibility issues we encounter with the content recommendation-enhanced landing page.
